components/service/firefox-accounts/src/test/java/mozilla/components/service/fxa/FxaDeviceConstellationTest.kt

```diff
@@ -232,6 +239,68 @@ class FxaDeviceConstellationTest {
         assertEquals(ConstellationState(currentDeviceExpired.into(), listOf(testDevice2.into())), constellation.state())
     }
 
+    @Test
+    @ExperimentalCoroutinesApi
+    fun `polling for commands with real account`() = runBlockingTest {
+        val testScope = GlobalScope
+
+        testScope.launch {
+            // Initialize things so the megazord is happy
+            WorkManagerTestInitHelper.initializeTestWorkManager(testContext)
+            Megazord.init()
+
+            // Create a test account
+            val testAccount = spy(NativeFirefoxAccount(
+                    handle = Long.MIN_VALUE,
+                    persistCallback = null
+            ))
+
+            // Create out device constellation object that accepts the account
+            val testConstellation = FxaDeviceConstellation(
+                    account = testAccount,
+                    scope = CoroutineScope(coroutinesTestRule.testDispatcher) + SupervisorJob()
+            )
+
+            testConstellation.pollForCommandsAsync().await()
+
+            verify(testAccount.pollDeviceCommands())
+        }
+    }
+
+    @Test
+    fun `test sending one tab`() = runBlockingTest {
+        // Create a scope to run this test inside
+        val testScope = GlobalScope
+
+        testScope.launch {
+            // Initialize things so the megazord is happy
+            WorkManagerTestInitHelper.initializeTestWorkManager(testContext)
+            Megazord.init()
+
+            // Create a test account
+            val testAccount = spy(NativeFirefoxAccount(
+                    handle = Long.MIN_VALUE,
+                    persistCallback = null
+            ))
+
+            // Create our device constellation object that accepts the account
+            val testConstellation = FxaDeviceConstellation(
+                    account = testAccount,
+                    scope = CoroutineScope(coroutinesTestRule.testDispatcher) + SupervisorJob()
+            )
+            val targetDeviceId = "testDevice"
+            val title = "Mozilla"
+            val url = "https://mozilla.org"
+            val response = testConstellation.sendCommandToDeviceAsync(
+                    targetDeviceId = targetDeviceId,
+                    outgoingCommand = DeviceCommandOutgoing.SendTab(title, url)
+            )
+
+            assertNotNull(response)
+            verify(testAccount.sendSingleTab(targetDeviceId, title, url))
```

Comment:
You're note awaiting for a response to return, so I'm not entirely sure how this is supposed to work...